### PR TITLE
Create app

### PR DESCRIPTION
**Pull request by Tshobohwa**
# FEATURE ADDED 
- [ ] Creation of the user interface of the app.

### CODE REVIEW

File: src/components/Keyboard.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      </button>
+      <button className="calculator-btn" type="button">
+        +/-
+      </button>
+      <button className="calculator-btn" type="button">
+        %
+      </button>
+      <button className="calculator-btn" type="button">
+        &div;
+      </button>
```
Comment (AnselemOdims) [inline src/components/Keyboard.js]: - [ ] Please kindly ensure that your design follows the color of the buttons in the original design
- [ ] Also, the division operator does not display as it should. See image below for reference

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Tshobohwa , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Tshobohwa** [comment]
- [ ] Fixed colors and displayed division sign.

**Review by julie-ify — APPROVED** [review]
Hi @Tshobohwa,

## Highlights!
- Replication of design ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**